gflownet/envs/lattice_parameters.py

```diff
@@ -67,17 +66,14 @@ def __init__(
             Note that it has to be defined in such a way that 90 and 120 degree angles will be present in the
             mapping from grid cells to angles (np.linspace(min_angle, max_angle, grid_size)).
 
-        min_step_len : int
-            Minimum value of the step (how many cells can be incremented in a single step).
-
-        max_step_len : int
-            Maximum value of the step (how many cells can be incremented in a single step).
+        max_increment : int
+            Maximum increment of each dimension by the actions.
         """
         super().__init__(
             n_dim=6,
             length=grid_size,
-            min_step_len=min_step_len,
-            max_step_len=max_step_len,
+            max_increment=max_increment,
+            max_dim_per_action=3,
```

Comment:
So `max_dim_per_action` will be fixed and equal to 3?

Reply:
In practice, yes, since we only allow to either the angles or the lengths to be changed in a single step, not both. It shouldn't really matter what we set this to, because it's only used in the Grid's `get_action_space`, I set it to 3 because it seemed more consistent to me (and in case of any future changes in the Grid).